challenge-03/README.md

```diff
@@ -150,8 +178,38 @@ deve conter no retorno da frase acima é "metro" no lugar de "metros".
 método), que será concatenada com a frase de retorno, mostrando a resposta
 correta, de acordo com os dados inseridos no objeto.
 */
-?
+pessoa.apresentacao = function() {
+
+	/*
+	Nesse tipo de validação eu usaria ternária:
+	
+	var genero = ( pessoa.sexo === 'Feminino' ) ? 'a' : 'o';
+	
+	Porém, seguindo o exercício, vai a forma proposta:
+	*/
+
+	if ( pessoa.sexo === 'Feminino' ) {
+		var genero = 'a';
+	} else {
+		var genero = 'o';
+	}
+	
+	if ( pessoa.idade === 1 ) {
+		var condicaoAno = '';
+	} else {
+		var condicaoAno = 's';
+	}
+	
+	if ( pessoa.caminhouQuantosMetros === 1 ) {
+		var condicaoCaminhou = '';
+	} else {
+		var condicaoCaminhou = 's';
+	}
```

Comment:
Uma outra forma que você poderia usar para validar seria declarar a variável no topo da função, com o valor padrão, e usar o if só para modificar, se a condição for verdadeira ;)